packages/create-cloudflare/src/helpers/args.ts

```diff
@@ -230,6 +249,22 @@ export const parseArgs = async (argv: string[]): Promise<Partial<C3Args>> => {
 	const additionalArgs =
 		doubleDashesIdx < 0 ? [] : argv.slice(doubleDashesIdx + 1);
 
+	const c3positionalArgs = c3Args.filter((arg) => !arg.startsWith("-"));
+
+	if (c3positionalArgs[2] === "telemetry") {
+		const action = c3positionalArgs[3];
+
+		switch (action) {
+			case "enable":
+			case "disable":
+			case "status":
+				return {
+					type: "telemetry",
+					action,
+				};
+		}
+	}
```

Comment:
It looks like if you run `npm create cloudflare telemetry something` that this will fallthrough to treating "telemetry" as the project name in the regular c3 flow

I think we should not fallthrough and instead throw an invalid usage error, explaining the only subcommands are enable, disable and status.